lib/util.js

```diff
@@ -103,27 +75,6 @@ export function publicKeyDerEncode({publicKeyBytes}) {
   // add a zero byte to the front of the publicKeyBytes, this results in
   // the bitstring being 256 bits vs. 170 bits (without padding)
   const zeroBuffer = Buffer.from(new Uint8Array([0]));
-  const keyBuffer = new ByteBuffer(Buffer.concat([zeroBuffer, publicKeyBytes]));
-
-  const a = asn1.create(
-    asn1.Class.UNIVERSAL,
-    asn1.Type.SEQUENCE,
-    true, [
-      asn1.create(asn1.Class.UNIVERSAL, asn1.Type.SEQUENCE, true, [
-        asn1.create(
-          asn1.Class.UNIVERSAL,
-          asn1.Type.OID,
-          false,
-          asn1.oidToDer(oids.EdDSA25519).getBytes()
-        ),
-      ]),
-      // public key
-      asn1.create(
-        asn1.Class.UNIVERSAL, asn1.Type.BITSTRING, false,
-        keyBuffer.getBytes()),
-    ]
-  );
-
-  const publicKeyDer = asn1.toDer(a);
-  return Buffer.from(publicKeyDer.getBytes(), 'binary');
+  const keyBuffer = Buffer.concat([zeroBuffer, publicKeyBytes]);
+  return Buffer.concat([DER_PUBLIC_KEY_PREFIX, keyBuffer]);
```

Comment:
The `zeroBuffer` was probably related to BITSTRING/DER formatting -- which should no longer be necessary since we have a full prefix. It may even be problematic. Can we remove it?

Reply:
removing it results in:

```
     Error: error:0D07209B:asn1 encoding routines:ASN1_get_object:too long
       fails if signing data is changed:
     Error: error:0D07209B:asn1 encoding routines:ASN1_get_object:too long
      at createPublicKey (internal/crypto/keys.js:345:10)
      at Object.verify (lib/crypto.js:77:29)
      at Object.verify (lib/Ed25519VerificationKey2020.js:240:23)
      at Context.<anonymous> (test/sign-verify.spec.js:38:35)
```

so we could add it to the prefix and cut out a step.

Reply:
See my suggestion above to add this zero byte onto the end of the prefix constant.

Reply:
Addressed herE: https://github.com/digitalbazaar/ed25519-verification-key-2020/pull/5/commits/c76e618e9d4d8866bd83b1f468a8f4ace5980944